src/app/pages/account/groups/group-form/group-form.component.ts

```diff
@@ -68,8 +77,19 @@ export class GroupFormComponent implements OnInit {
 
   ngOnInit(): void {
     this.setupForm();
+    this.getInitialPrivileges();
   }
 
+  readonly privilegesProvider: ChipsProvider = (query: string) => {
+    return this.ws.call('privilege.query', []).pipe(
```

Comment:
This query should be sent to middleware. Instead of being applied locally once the response has been fetched. Otherwise, we don't need to make the same request and fetch the same results every time query changes.

Reply:
@RehanY147  - what do you mean?
this is the provider for chips component?

Reply:
> This query should be sent to middleware. Instead of being applied locally once the response has been fetched. Otherwise, we don't need to make the same request and fetch the same results every time query changes.

we have the same for privileges form, we need to grab all the items once and filter locally I suppose